README.md

```diff
@@ -1,21 +1,61 @@
-# javascript-utilities
-
-[![CircleCI](https://circleci.com/gh/Shopify/javascript-utilities.svg?style=svg)](https://circleci.com/gh/Shopify/javascript-utilities)
-
-A set of utilities for JavaScript and TypeScript projects at Shopify.
+<h1 align="center">
+  <br>
+  <a href="http://www.shopify.com/"><img src="https://cdn.shopify.com/assets2/press/brand/shopify-logo-main-small-f029fcaf14649a054509f6790ce2ce94d1f1c037b4015b4f106c5a67ab033f5b.png" alt="Shopify" width="200" /></a>
```

Comment:
Really don’t like the big logo at the top, if we want to include it I think it should be below the license.